pkg/monitortests/node/kubeletselinuxlabels/monitortest.go

```diff
@@ -0,0 +1,166 @@
+package kubeletselinuxlabels
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/openshift/origin/pkg/monitortestframework"
+	"github.com/openshift/origin/pkg/monitortestlibrary/statetracker"
+	exutil "github.com/openshift/origin/test/extended/util"
+
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+	"github.com/openshift/origin/pkg/test/ginkgo/junitapi"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	msgPhaseDrain       = "drained node"
+	msgPhaseOSUpdate    = "updated operating system"
+	msgPhaseReboot      = "rebooted and kubelet started"
+	testName            = "[sig-node][kubelet] selinux labels on kubelet process should always be kubelet_t"
+	kubeletProcessLabel = "kubelet_t"
+	kubeletBinaryLabel  = "kubelet_exec_t"
+)
+
```

Comment:
Let's add a comment on what we are testing and why. ie: the bash scripts might have different selinux labels.

Reply:
added a comment below.